projects/noloan/app/filter/SpamFilter.java

```diff
@@ -4,20 +4,32 @@
 
 import com.google.common.io.Resources;
 import com.google.common.collect.ImmutableList;
+
```

Comment:
How about if the spam filter adds the probability to the SmsMessage, for debugging?

Reply:
@oferb won't it break backwards compat with `.prototxt` we have?

Reply:
@oferb we could `Log` it instead